.gitignore

```diff
@@ -101,3 +101,6 @@ deps/
 android-toolchain-arm/
 cc-toolchain/
 /TraceOutput.txt
+Build/VcBuild
```

Comment:
that was directories, they are files in my change. I'm creating symbolic link to my SSD